### PR TITLE
ER-483: Fixed autocomplete hack

### PR DESCRIPTION
**Pull request by rimi-itk**

### CODE REVIEW

File: profiles/ding2/modules/opensearch/includes/opensearch.search.inc [hunk 1]
```diff
@@ -691,19 +676,14 @@ function opensearch_search_autocomplete($string) {
...
-  }
+  // Hack to use old suggestions service.
+  $query += variable_get('ting_search_autocomplete_settings', array());
+  $query['query'] = $query['q'];
+  $query['maxSuggestions'] = $query['rows'];
+
+  $method = variable_get('ting_search_autocomplete_method', 'facets');
 
-  $url = url($rest_url, array('query' => $query));
+  $url = url('http://opensuggestion.addi.dk/b3.5_2.0/rest/'.$method, array('query' => $query));
```
Comment (cableman) [inline profiles/ding2/modules/opensearch/includes/opensearch.search.inc]: Should use the URL set in the admin and store in: `variable_get('opensearch_search_autocomplete_suggestion_url', '');`

File: sites/all/patches/opensearch-autocomplete.patch [hunk 2]
```diff
@@ -55,33 +41,20 @@ index ff141511..0cbe9a3c 100644
...
 +  // Hack to use old suggestions service.
 +  $query += variable_get('ting_search_autocomplete_settings', array());
 +  $query['query'] = $query['q'];
 +  $query['maxSuggestions'] = $query['rows'];
 +
 +  $method = variable_get('ting_search_autocomplete_method', 'facets');
-+
+ 
+-  $url = url($rest_url, array('query' => $query));
 +  $url = url('http://opensuggestion.addi.dk/b3.5_2.0/rest/'.$method, array('query' => $query));
```
Comment (cableman) [inline sites/all/patches/opensearch-autocomplete.patch]: Should use the URL set in the admin and store in: `variable_get('opensearch_search_autocomplete_suggestion_url', '');`

File: profiles/ding2/modules/opensearch/includes/opensearch.search.inc (hunk 1 above)
Comment (cableman) [reply profiles/ding2/modules/opensearch/includes/opensearch.search.inc]: Which is set to `http://opensuggestion.addi.dk/b3.5_2.0`

Comment (rimi-itk) [reply profiles/ding2/modules/opensearch/includes/opensearch.search.inc]: Well, this is a hack and it may make sense to hardcode the url …

Comment (cableman) [reply profiles/ding2/modules/opensearch/includes/opensearch.search.inc]: Then the hack should remove the option to set the URL in the admin UI. It don't make sens to have an UI to set the URL which have not effect.

**Pull request merged**